### ISSUE TITLE
tests are not running

### ISSUE THREAD

**Issue by kdaily** [Sage-Bionetworks/annotator#52]
```
(annotator) [kdaily@tigger annotator]$ nosetests

----------------------------------------------------------------------
Ran 0 tests in 0.535s

OK
(annotator) [kdaily@tigger annotator]$ pytest 
============================================================================ test session starts =============================================================================
platform linux2 -- Python 2.7.14, pytest-3.4.0, py-1.5.2, pluggy-0.6.0
rootdir: /home/kdaily/src/annotator, inifile:
collected 0 items                                                                                                                                                            

======================================================================== no tests ran in 6.02 seconds ========================================================================
```

**Comment by teslajoy** [comment]
runs via `nosetests -vs tests/integration/test_json2synapse.py:check_keys`
But I'll have to change it to pytest.

**Comment by kdaily** [comment]
OK! need to figure out how to register it in the appropriate way so that when new tests are added, all tests can be run via just `pytest`. Thanks!

**Comment by philerooski** [comment]
Tests fail for me unless I run them in the directory they live in. I think the root of the problem is the relative import of tests/sampleFile.csv and tests/sampleMeta.csv. Should we just include the contents of these files in conftest.py? I don't foresee them getting very large.

**Comment by teslajoy** [comment]
We need to make a conversion _to not push code_ to master if the tests fail on py27 and py3 
<img width="1110" alt="screen shot 2018-02-04 at 11 38 35 am" src="https://user-images.githubusercontent.com/2817496/35781527-3437cca2-09a0-11e8-8d42-6df5c42a9d0a.png">

**Comment by philerooski** [comment]
I strongly agree, of course. Luckily for now it's just a file dependency issue and not the actual methods breaking. I'll work on getting that fixed today.

@teslajoy Is the test project at a point where it can be "set in stone" for consistent testing or should we just pull SAMPLE_FILE and SAMPLE_META from Github for now?

**Comment by teslajoy** [comment]
Pulling from github's master for now.

**Comment by teslajoy** [comment]
![screen shot 2018-02-16 at 11 19 33 am](https://user-images.githubusercontent.com/2817496/36325059-54679c7c-130b-11e8-86ed-976a102c9caf.png)

Much better, just have to reopen this issue since not all tests are passing yet.

**Comment by philerooski** [comment]
It looks like the tests are all passing in python 3 but the above tests fail for python2.7

**Comment by teslajoy** [comment]
I tested two hypothesis: 
scope issues with pytest conftest:  I can get the syn, entities, sampleFiles, etc. so this is not the issue. 
incompatibility of pandas and pytest version: not true either. I see the df 

Next up:
uploading a file to synapse fails and returns HTTP 200 
instead of ERROR we get FAILED via the pandas.testing for I think the files can't be stored and df doesn't exists. 

not sure why in py3 upload to synapse works.
